guides/source/active_record_querying.md

```diff
@@ -673,7 +673,7 @@ GROUP BY date(created_at)
 HAVING sum(price) > 100
 ```
 
-This will return single order objects for each day, but only those that are ordered more than $100 in a day.
+This will return single order objects for each day, but only days in which more than $100 was ordered.
```

Comment:
Yea I agree this is confusing. How about:

> This returns the date and total price for each order object, grouped by the day they were ordered and where the price is more than $100.

Otherwise this is good, so if you make that change and squash, I'll merge. Thanks! :+1: